server/sonar-server/src/test/java/org/sonar/server/permission/ws/AddUserActionTest.java

```diff
@@ -246,6 +247,10 @@ public void adding_project_permission_is_allowed_to_project_administrators() thr
   }
 
   private void loginAsAdmin() {
-    userSession.login().setGlobalPermissions(SYSTEM_ADMIN);
+    loginAsOrganizationAdmin(db.getDefaultOrganization());
+  }
+
+  private void loginAsOrganizationAdmin(OrganizationDto org) {
```

Comment:
isn't a test missing making use of this method with an organization which is not the default one?